exercises/roman-numerals/description.md

```diff
@@ -23,8 +21,7 @@ into stone tablets).
 The maximum number supported by this notation is 3,999.
 (The Romans themselves didn't tend to go any higher)
 
-Wikipedia says: Modern Roman numerals ... are written by expressing each
-digit separately starting with the left most digit and skipping any
+Wikipedia says: Modern Roman numerals ... are written by expressing each digit separately starting with the left most digit and skipping any
```

Comment:
line ends before sentence end

Reply:
This one was tricky. The line ended just at the edge of my terminal and so I didn't notice that I had a line break instead of a line wrap. Fixing!